spec/controllers/datasets_controller_spec.rb

```diff
@@ -10,21 +10,52 @@
     it "handles the index page" do
       sign_in user
       get :index
-      expect(response.status).to eq(200)
+      expect(response).to render_template("index")
```

Comment:
The gem `rails-controller-testing` that I added on this PR allows to check that the proper view is rendered rather than just an HTTP 200 (I stole this from Orangelight, I think)

Reply:
💜